test/Limiters/limiter.jl

```diff
@@ -113,23 +128,25 @@ end
         limiter = Limiters.QuasiMonotoneLimiter(ρq)
         Limiters.compute_bounds!(limiter, ρq, ρ)
 
+        is_cpu = device isa ClimaComms.AbstractCPUDevice
         for h2 in 1:n2
             for h1 in 1:n1
                 s = slab(limiter.q_bounds, h1 + n1 * (h2 - 1))
-                q_min = s[1]
-                q_max = s[2]
-                @test q_min.x ≈ 2 * (h1 - 1)
-                @test q_min.y ≈ 3 * (h2 - 1)
-                @test q_max.x ≈ 2 * h1
-                @test q_max.y ≈ 3 * h2
+                # @show s
```

Comment:
Is this a stray debugging comment?

Reply:
Ah, I thought I rebased this away, I'll fix this